core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1006,23 +1007,29 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
                         )
                       ),
                       attributesEnabled,
-                      Some(wsmContext.getCloudPlatform)
+                      wsmContext.getCloudPlatform
                     )
                   // remove submission stats if they were not requested
                   val submissionStats: Option[WorkspaceSubmissionStats] = if (submissionStatsEnabled) {
                     Option(submissionSummaryStats(wsId))
                   } else {
                     None
                   }
-                  Option(
-                    WorkspaceListResponse(
-                      accessLevel,
-                      workspaceDetails,
-                      submissionStats,
-                      workspaceSamResource.public.roles.nonEmpty || workspaceSamResource.public.actions.nonEmpty,
-                      Some(wsmContext.policies)
-                    )
-                  )
+                  // Remove workspaces that are non-ready with no cloud context (Ready workspaces with no
```

Comment:
This is to maintain our previous behavior… though I am starting to wonder if we could just allow workspaces with no cloud context to be returned. However, we would need to check that the list UI view can handle the cloud context being None, and think through if there are any other ramifications.

Reply:
Created https://broadworkbench.atlassian.net/browse/WOR-1314